File: Mundial.Infra/Model/File.cs
using System;

namespace Mundial.Infra.Model
{
    public class File: MundialModel
    {
        public DateTime CreationFileDate {get; set;}

        public string Name { get; set; }

        // public int? CustomerAddressId {get; set;}

        public string City {get; set;}

        public string Street {get; set;}

        public string Neighborhood {get; set;}

        public int? HouseNumber {get; set;}

        public string Complement {get; set;}

        public string CEP {get; set;}
        
        // public Address CustomerAddress {get; set;}

        public string PhoneNumber {get; set;}

        public string Profession {get; set;}

        public float? Salary {get; set;}

        // public int? WorkAddressId {get; set;}

        public string WorkCity {get; set;}

        public string WorkStreet {get; set;}

        public string WorkNeighborhood {get; set;}

        public int? WorkNumber {get; set;}

        public string WorkComplement {get; set;}

        public string WorkCEP {get; set;}

        // public Address WorkAddress {get; set;}
     
        public int? CPF {get; set;}

        public string RG {get; set;}

        public string Work {get; set;}

        public DateTime? DateOfBirth { get; set; }

        public string PartnerName {get; set;}

        public string PartnerWork {get; set;}

        public string PartnerPhone {get; set;}

        public float? PartnerSalary {get; set;}

        public string ParentsObservation {get; set;}

        public string OthersStores {get; set;}

        public string FriendName {get; set;}

        public string FriendsPhone {get; set;}

        public string FileObservation {get; set;}

    }
}

File: Client/mundial-client/src/app/models/file.ts
import { MundialModel } from './abstract/mundial-model';

export class File extends MundialModel {

  public CreationFileDate: Date;

  public Name: string;

  // public CustomerAddressId: number;

  // public CustomerAddress? Address;
  public City?: string;

  public Street?: string; 

  public Neighborhood?: string; 

  public HouseNumber?: number; 

  public Complement?: string; 

  public CEP?: string;

  public PhoneNumber?: string;

  public Profession?: string;

  public Salary?: number;

  // public WorkAddressId?: number;

  // public WorkAddress?: Address;

  public WorkCity?: string;

  public WorkStreet?: string; 

  public WorkNeighborhood?: string; 

  public WorkNumber?: number; 

  public WorkComplement?: string; 

  public WorkCEP?: string;

  public CPF?: number;

  public RG?: string;

  public Work?: string;

  public DateOfBirth?: Date;

  public PartnerName?: string;

  public PartnerWork?: string;

  public PartnerPhone?: string;

  public PartnerSalary?: number;

  public ParentsObservation?: string;

  public OthersStores?: string;

  public FriendName?: string;

  public FriendsPhone?: string;

  public FileObservation?: string;

  constructor(
    Id: number,
    CreationDate: Date,
    ExclusionDate: Date,
    Number: number,
    CreationFileDate: Date,
    Name: string,
    // CustomerAddressId?: number,
    // CustomerAddress?: Address,
    City?: string,
    Street?: string,   
    Neighborhood?: string,  
    HouseNumber?: number,   
    Complement?: string,
    CEP?: string,
    PhoneNumber?: string,
    Profession?: string,
    Salary?: number,
    // WorkAddressId?: number,
    // WorkAddress?: Address,
    WorkCity?: string,
    WorkStreet?: string,   
    WorkNeighborhood?: string,  
    WorkNumber?: number,   
    WorkComplement?: string,
    WorkCEP?: string,
    CPF?: number,
    RG?: string,
    Work?: string,
    DateOfBirth?: Date,
    PartnerName?: string,
    PartnerWork?: string,
    PartnerPhone?: string,
    PartnerSalary?: number,
    ParentsObservation?: string,
    OthersStores?: string,
    FriendName?: string,
    FriendsPhone?: string,
    FileObservation?: string
  ) {
    super(CreationDate, Number, ExclusionDate,Id);

    this.Number = Number;

    this.CreationFileDate = CreationFileDate;

    this.Name = Name;

    // this.CustomerAddressId = CustomerAddressId;

    // this.CustomerAddress = CustomerAddress;

    this.City = City;

    this.Street = Street; 

    this.Neighborhood = Neighborhood;  

    this.HouseNumber = HouseNumber; 

    this.Complement = Complement;

    this.CEP = CEP;

    this.PhoneNumber = PhoneNumber;

    this.Profession = Profession;

    this.Salary = Salary;

    // this.WorkAddressId = WorkAddressId;

    // this.WorkAddress = WorkAddress;

    this.WorkCity = WorkCity;

    this.WorkStreet = WorkStreet; 

    this.WorkNeighborhood = WorkNeighborhood;  

    this.WorkNumber = WorkNumber; 

    this.WorkComplement = WorkComplement;

    this.WorkCEP = WorkCEP;

    this.CPF = CPF;

    this.RG = RG;

    this.Work = Work;

    this.DateOfBirth = DateOfBirth;

    this.PartnerName = PartnerName;

    this.PartnerWork = PartnerWork;

    this.PartnerPhone = PartnerPhone;

    this.PartnerSalary = PartnerSalary;

    this.ParentsObservation = ParentsObservation;

    this.OthersStores = OthersStores;

    this.FriendName = FriendName;

    this.FriendsPhone = FriendsPhone;

    this.FileObservation = FileObservation;
  }

  
}
